start_session.php

```diff
@@ -126,6 +126,26 @@
             ]
         ],
     ],
+    'irmatube_premium' => [
+        'nextSession' => [
+            'url' => IRMATUBE_NEXT_SESSION_URL
+        ],
+        'request' => [
+            '@context' => 'https://irma.app/ld/request/disclosure/v2',
+            'disclose' => [
+                [
+                    [ISSUER .'.pbdf.facebook.fullname'],
```

Comment:
It would be better if you use the `linkedin` credential here as the `facebook` credential is no longer obtainable.